### DISCUSSION TITLE
Generators

### DISCUSSION THREAD

**Discussion by echodot1** [BG-Software-LLC/SuperiorSkyblock2#565] [Pending Feature Requests] [answered]
My suggestion is - ability to place water in the nether so players can have nether generators. and can gain access to nether blocks without having the buy them and end generators, Also maybe a idea the higher the level of the island gets the more ores you can reseive in your generator

**Comment by Lemvs** [comment, marked as answer]
In version 1.16+, a basalt generator was added that can be built in the Nether, allowing for the configuration of generator rates for that dimension. Water cannot be placed in the Nether.

@OmerBenGera you can mark this answered and close it.
